### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by ydah**
This PR is fix some spelling.

- s/platfoms/platforms/
- s/directove/directive/
- s/colmn/column/
- s/DELIMETER/DELIMITER/
- s/contant/constant/
- s/ambigious/ambiguous/
- s/interporation/interpolation/
- s/hightlight/highlight/
- s/specifing/specifying/
- s/indendation/indentation/
- s/fileand/file and/
- s/dedented/indented/
- s/ofmodule/of module/
- s/argumnets/arguments/
- s/alignmenton/alignment on/
- s/bracketwith/bracket with/
- s/correcs/corrects/
- s/accpets/accepts/
- s/condtionals/conditionals/
- s/offsense/offense/
- s/brancheswith/branches with/
- s/branchsi/branches/
- s/consntant/constant/
- s/wheh/when/
- s/wtyle/style/
- s/condidate/candidate/
- s/arguemnt/argument/
- s/offens/offense/
- s/offfense/offense/
- s/condtion/condition/
- s/rescueing/rescuing/
- s/commnet/comment/
- s/parameteres/parameters/
- s/lodable/loadable/

-----------------

Before submitting the PR make sure the following are checked:

* [x] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences.
* [x] Wrote [good commit messages][1].
* [-] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [-] Added tests.
* [x] Ran `bundle exec rake default`. It executes all tests and runs RuboCop on its own code.
* [-] Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

[1]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by koic** [comment]
> I think it is difficult to see the log if I squash because there are many modifications, so I have separated the commits but will squash if necessary.

I won't look at each commit (all spelling mistakes corrected), please squash it.

**Comment by koic** [comment]
And I think it would be better to update the spell checking workflow of GitHub Actions so that this can be detected.
See: https://github.com/rubocop/rubocop/pull/10128

File: .github/workflows/spell_checking.yml [hunk 1]
```diff
@@ -30,4 +30,4 @@ jobs:
       - name: Install
         run: wget -O - -q https://raw.githubusercontent.com/client9/misspell/master/install-misspell.sh | sh -s -- -b .
       - name: Misspell
-        run: git ls-files --empty-directory | xargs ./misspell -i 'enviromnent' -error
+        run: git ls-files --empty-directory | xargs ./misspell -i 'enviromnent,specifing,accpets,parameteres' -error
```
Comment (koic) [inline .github/workflows/spell_checking.yml:33]: Why do this PR need this update?

File: codespell.txt [hunk 2]
```diff
@@ -7,3 +7,37 @@ irregardless
...
+wtyle
+condidate
+arguemnt
+offens
+offfense
+condtion
+rescueing
+commnet
+parameteres
+lodable
```
Comment (koic) [inline codespell.txt:43]: Ditto.

File: .github/workflows/spell_checking.yml (hunk 1 above)
Comment (ydah) [reply .github/workflows/spell_checking.yml]: I was mistaken. I removed it.

**Comment by ydah** [comment]
I sent a patch to add it to the dictionary.
https://github.com/codespell-project/codespell/pull/2467

**Comment by bbatsov** [comment]
Nice! How did you manage to identify all those typos?

**Comment by ydah** [comment]
I often use VSCode and the extension [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) to detect and fix them when I find them in the typo fix branch. Once I have accumulated a few, I often check for similar typos in [cspell](https://www.npmjs.com/package/cspell) and put them together in a PR.

**Pull request merged**